Fix extra whitespace below media embeds. And a minor resizing issue.

schwers-zz opened the pull request:
Fixes a visual bug in media embeds having extraneous whitespace before
the author. This is really common on Twitter embeds.

https://reddit.atlassian.net/browse/WEB-1217

Also fixes an issue with the embed shrinking in width when the
page is resized, and never growing back to its proper size.

Tested with various Twitter and Imgur embeds.

Some threads on my staging
[SuperSmash](https://schwers.staging.wireddit.com/live/tsdh1ylqjv3b)
[Pluto](https://schwers.staging.wireddit.com/live/v8j2tqin01cf)
[Paris](https://schwers.staging.wireddit.com/live/vwwnkuplwr9y)

:eyeglasses: @florenceyeun @umbrae @spez

umbrae commented on reddit_liveupdate/templates/liveupdatemediaembedbody.html:
```diff
@@ -45,7 +45,7 @@
             /* Currently just check on every set of mutations. */
             var observer = new MutationObserver(checkDimensions),
-                config = { attributes: true, subtree: true};
+                config = { attributes: true, subtree: true, childList: true};
```
:no_bell: I'm surprised that `subtree` doesn't also take into account all direct children already - does this do something different?

schwers replied:
I was surprised too, it seems like `subtree` and `attributes` just checks if props of things change. I needed `childList` to make the observer fire for plain add/removes. Twitter embeds are an example of this, without `childList` the observer isn't fired when their placeholder blockquote is removed

umbrae replied:
Ahh, TIL!

spladug commented:
Could you split the "Also fixes..." stuff out into a separate commit please?

https://github.com/reddit/styleguide/tree/master/general-style-guidelines#commits

spez commented:
:fish:

Next time can you include screenshots? I'm just screening UI

schwers committed:
Fix media embeds height.

This is really common on Twitter embeds

https://reddit.atlassian.net/browse/WEB-1217

schwers committed:
Fix width of media embeds after resizing

When resized embeds weren’t growing back to the proper width

umbrae commented:
@schwers We're a little pickier than the average bear on commit messages at reddit, do you think for your commits you could make the title a short, active sentence that fits without ellipsizing? (< 65 chars I think). This makes it easy to scan when you're going back in history and trying to figure out what happened.

:nail_care: looks great other than that! I'm having trouble getting to staging since upgrading my machine but the code looks good.

schwers commented:
@spladug @umbrae these better?

@spez Sure, I can add some screenshots for you and Chris. Feel free to come check it on my machine as well

umbrae commented:
:fish: Yep!

On Friday, November 20, 2015, Ryan Schwers <notifications@github.com> wrote:

> @spladug <https://github.com/spladug> @umbrae <https://github.com/umbrae>
> these better?
>
> @spez <https://github.com/spez> Sure, I can add some screenshots for you
> and Chris. Feel free to come check it on my machine as well
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/reddit/reddit-plugin-liveupdate/pull/122#issuecomment-158563075>
> .
>

spladug commented:
:fish: thanks!

schwers commented:
closed via `40d825408d24481235acf1f5344051cc46407a01`
